Vizzuality/half-earth-v3
# feat(front): onboarding interaction [HE-328]

## Onboarding interaction
- [ ] Block user interaction with veil
- [x] Store waitingInteraction on UI store ('waitingInteraction' = true | false)
- [x] Allow user interaction when waitingInteraction is true (add an attribute to the div to click to allow pointer-events and allow scroll when needed)
- [ ] Add tooltip: "Click here to continue"
- [x] Change section stored on UI after the user has made the interaction and set waitingInteraction to false

### Designs
[Figma Prototype](https://www.figma.com/proto/2YAp3AS5lYCj2IWEAN8AJw/HE---Map?node-id=272%3A63999&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=4%3A15413&show-proto-sidebar=1)

### Testing instructions
Please, change the `onBoardingType` and `onBoardingStep` on `containers/landing/hero/hero-component.jsx` to see behaviour until it's merged with the sound feature.

### Feature relevant tickets
[HE-328](https://vizzuality.atlassian.net/browse/HE-328)
